src/wp-admin/update-core.php

```diff
@@ -1027,7 +1027,7 @@ function do_undismiss_core_update() {
 	?>
 	<div class="wrap">
 	<h1><?php _e( 'WordPress Updates' ); ?></h1>
-	<p><?php _e( 'Here you can find information about updates, set auto-updates and see what plugins or themes need updating.' ); ?></p>
+	<p><?php _e( 'Here you can find information about pending updates of plugins, themes and translations. You can also change if you want WordPress to update automatically to the latest major version or only receive minor updates of the currently installed version.' ); ?></p>
```

Comment:
```suggestion
	<p><?php _e( 'Here you can find information about pending updates of WordPress, plugins, themes and translations. You can also choose whether to enable automatic updates for all new versions of WordPress, or for maintenance and security releases only.' ); ?></p>
```
This suggestion:
- Adds "WordPress" to the list of "pending updates", as manual updates of Core are possible here.
- Makes the "automatic updates" text reflect the links that appears further down:
> Enable automatic updates for all new versions of WordPress.

> Switch to automatic updates for maintenance and security releases only.